README.md

```diff
@@ -2,32 +2,48 @@
+
 ## Getting started with the Template BPA
 The Template BPA is built using .NET 5.  Having the [.NET 5 Runtime](https://dotnet.microsoft.com/download) installed is currently a prerequisite.
 
-After ensuring the .NET Runtime is installed, download the latest Template BPA release by clicking on 'Releases' to the right.
+After ensuring the .NET Runtime is installed, download the latest Template BPA release in [the releases section](https://github.com/Azure/template-analyzer/releases).
 
-Also explore [information about built-in rules](docs\built-in-bpa-rules.md).
+To preview the rules that come bundled with the Template BPA, explore [the built-in rules](docs/built-in-bpa-rules.md).
 
 ## Using the Template BPA
 The Template BPA is executed via a command line.  Here is the format to invoke it:
 
 `TemplateAnalyzer.exe -t <template-path> [-p <parameters-path>]`
 
 ### Input
-The Template BPA depends on a few inputs – an ARM template to analyze, ARM template parameters (optional), and the Template BPA rules (not specified on the command line). The Template BPA rules are written in JSON, located in *Rules/BuiltInRules.json* (starting from the same directory *TemplateAnalyzer.exe* is in). You can add to and/or modify that file to change the rules that are run. See the [documentation for more information about how to author Template BPA JSON rules](./docs/authoring-json-rules.md). The Template BPA then runs the configured rules against the provided [ARM templates](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/) and their corresponding [template parameters](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files) if specified. If no template parameters are specified, then the Template BPA generates the minimum number of placeholder parameters to properly evaluate [template functions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions) in the ARM templates.
+The Template BPA depends on a couple inputs – an ARM template to analyze, specified with the `-t` or `--template-file-path` argument, and an (optional) ARM template parameters file, specified with the `-p` or `--parameters-file-path` argument. The Template BPA then runs the [configured rules](#understanding-and-customizing-rules) against the provided ARM template and its corresponding [template parameters](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files), if specified. If no template parameters are specified, then the Template BPA generates the minimum number of placeholder parameters to properly evaluate [template functions](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions) in the ARM template.
```

Comment:
Saying it "depends on" and then noting that parameters is optional seems strange to me. Instead maybe do something like 

The Template BPA allows for the following inputs:
- `-t` or `--template-file-path` - an ARM template to analyze
- (Optional) `-p` or `--parameters-file-path` - an ARM template parameters file